test/tools/libtesteth/boostTest.cpp

```diff
@@ -36,15 +36,17 @@ static std::ostringstream strCout;
 std::streambuf* oldCoutStreamBuf;
 std::streambuf* oldCerrStreamBuf;
 
-void createRandomTestWrapper()
+void customTestSuiteWrapper()
```

Comment:
Is this still being used only for creating random test?  In that case, I think the old name is better.